src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -23,14 +23,20 @@ const MEDIGAP = {
   medigapPlanM: 'Medigap Plan M',
 };
 
+function insuranceName(applicant, isPrimary) {
+  return isPrimary
+    ? applicant?.applicantPrimaryProvider
+    : applicant?.applicantSecondaryProvider;
+}
+
```

Comment:
This logic is going to get used in all the shared primary/secondary health schemas, so pulling out into standalone fn